### ISSUE TITLE
Make it responsive!

### ISSUE THREAD

**Issue by Brijesh097** [Brijesh097/sies-gst-library#2]
Make the **login.html** inside **login** folder **RESPONSIVE**

**Comment by Priyansh2** [comment]
@Brijesh097 you mean login page of your website ?

**Comment by bxrdy** [comment]
@Priyansh2 Yes!

**Comment by rowhitswami** [comment]
I'd love to take this one! :blush:

**Comment by bxrdy** [comment]
@rowhitswami Yes go ahead, make it **responsive** and send a **PR**

**Comment by rowhitswami** [comment]
@Brijesh097 Please review my PR #4  and see the screenshot of changes here : https://ibb.co/cBtYbG

**Comment by bxrdy** [comment]
@rowhitswami Thanks!

**Comment by reqyo** [comment]
Has this been completed?

**Comment by bxrdy** [comment]
@reqyo Yes! But if you want you can customize it a little bit and send a PR.

**Comment by rowhitswami** [comment]
@Brijesh097 I can't see my changes in the website, however, the PR is merged.

**Comment by bxrdy** [comment]
@rowhitswami Send the PR once again! There were some merge conflict issues.

**Comment by rowhitswami** [comment]
#5 See the PR here @Brijesh097

**Comment by reqyo** [comment]
Hey mate, I added viewheight on the main home page, so now it will scroll directly down from the jumbotron, do you want me to submit a pull request?

**Comment by bxrdy** [comment]
@reqyo Yes!

**Comment by AshBardhan** [comment]
@Brijesh097 Are you OK to have "Responsive Font Size" on your website?

**Comment by bxrdy** [comment]
@AshBardhan Yes go ahead!
